### PR TITLE
Calling the ./gradlew command from the same directory as where the script is placed

### PR DESCRIPTION
**Pull request by neomatrix369**

### CODE REVIEW

File: get_coverage_for_challenge.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ JACOCO_TEST_REPORT_XML_FILE="${SCRIPT_CURRENT_DIR}/build/reports/jacoco/test/jac
 mkdir -p ${SCRIPT_CURRENT_DIR}/target
 JAVA_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/target/java-code-coverage.txt"
 
-./gradlew -q clean test jacocoTestReport || true 1>&2
+( cd ${SCRIPT_CURRENT_DIR} && ./gradlew -q clean test jacocoTestReport || true 1>&2 )
```
Comment (julianghionoiu) [inline get_coverage_for_challenge.sh]: Run gradle with ${SCRIPT_CURRENT_DIR}/gradlew -p ${SCRIPT_CURRENT_DIR} insead of `cd`

File: get_coverage_for_challenge.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ JACOCO_TEST_REPORT_XML_FILE="${SCRIPT_CURRENT_DIR}/build/reports/jacoco/test/jac
 mkdir -p ${SCRIPT_CURRENT_DIR}/target
 JAVA_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/target/java-code-coverage.txt"
```
Comment (julianghionoiu) [inline get_coverage_for_challenge.sh]: The coverage info should be written into a different file. See test output for more info on what the expectation is.

Comment (neomatrix369) [reply get_coverage_for_challenge.sh]: Has this been changed? I know this worked previous, have you got this to work after your refactoring?

**Comment by neomatrix369** [comment]
We have a new issue when I run this via the container I get:
```
++ exec /usr/lib/jvm/java-8-openjdk-amd64/bin/java -Dorg.gradle.appname=get_coverage_for_challenge.sh -classpath /srv/tdl-runner/gradle/wrapper/gradle-wrapper.jar org.gradle.wrapper.GradleWrapperMain -q test jacocoTestReport

FAILURE: Build failed with an exception.

* What went wrong:
Task 'test' not found in root project 'srv'.

* Try:
Run get_coverage_for_challenge.sh tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
Test failed due to result mismatch
   Actual result: 'Set executable permissions for: /root/.gradle/wrapper/dists/gradle-2.2-all/1vevkra640w3rb9hkuw50q5we/gradle-2.2/bin/gradle'
   Expected result: '100'
```

When I run the tdl-runner-java from CLI I get no error or output.

Lets discuss this, this is strange and all of this worked when we merged these changes previously.

**Review by julianghionoiu — APPROVED** [review]

**Pull request merged**